master/buildbot/newsfragments/dockerworker.bugfix

```diff
@@ -0,0 +1 @@
+buildbot-worker docker image no longer use pidfile. This allows to autorestart a docker worker upon crash.
```

Comment:
This is the failure:

    relnotes/index.rst:19:autorestart:["auto restart", "auto-restart", "autostart", "autoregressive", "autorepeat", "authoress", "intrastate", "interested"]

I would hyphenate it: "auto-restart".